cmd/swarm/main.go

```diff
@@ -351,21 +352,61 @@ func registerBzzService(bzzconfig *bzzapi.Config, stack *node.Node) {
 	}
 }
 
-func getAccount(bzzaccount string, ctx *cli.Context, stack *node.Node) *ecdsa.PrivateKey {
-	//an account is mandatory
-	if bzzaccount == "" {
-		utils.Fatalf(SwarmErrNoBZZAccount)
-	}
-	// Try to load the arg as a hex key file.
-	if key, err := crypto.LoadECDSA(bzzaccount); err == nil {
-		log.Info("Swarm account key loaded", "address", crypto.PubkeyToAddress(key.PublicKey))
-		return key
+//getAccount returns the bzzaddress and the private key associated to that address
+func getAccount(bzzaccount string, ctx *cli.Context, stack *node.Node) (string, *ecdsa.PrivateKey) {
```

Comment:
Maybe rename to `getOrCreateAccount` and amend the godoc to explain that this will create an account if you don't have one and interactively ask for a password.